http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -166,10 +168,14 @@ protected String getRedirectUri(String state) {
 
             List<String> forwardableQueryParams = Arrays.asList(LOGIN_HINT, DOMAIN_HINT, KC_IDP_HINT, PROMPT, MAX_AGE, UI_LOCALES, SCOPE);
             List<NameValuePair> forwardedQueryParams = new ArrayList<>(forwardableQueryParams.size());
+            Set<String> allScopes = new HashSet<>();
+            if (deployment.getScope() != null && !deployment.getScope().isEmpty()){
+                allScopes.addAll(Arrays.asList(deployment.getScope().split("\\s+")));
+            }
```

Comment:
Since a similar if statement is also needed for the query param case, we could add a helper method, say `addScopes(String scope, HashSet<String> allScopes)` that adds the scopes from the given parameter to the given set. 

That way the new helper method could be called for both cases.